source/usage-examples.txt

```diff
@@ -48,22 +48,49 @@ or you can
 Once you have imported the dataset, you can copy and paste a usage
 example into your development environment of choice. You can follow the
 :doc:`quick start guide </quick-start>` to learn more about getting
-started with the MongoDB Go Driver. Once you've copied a usage example,
-you'll need to edit the connection string to get the example connected to
-your instance of MongoDB:
+started with the MongoDB Go Driver. 
+
+Once you've copied a usage example, you'll need to edit and place
+:ref:`your connection string <usage-examples-connection-string>` in
+:ref:`your environment variable <usage-examples-env-variable>` to get
+the example connected to your instance of MongoDB. 
```

Comment:
Change: 
- Use the simplest tense. [style guide ref](https://docs.mongodb.com/meta/style-guide/writing/use-effective-verbs/#use-the-simplest-tense)
- I think its understood that they will have copied and pasted (especially given the previous paragraph) and we can simplify this down.

```suggestion
To connect the example to your instance of MongoDB, you must edit
:ref:`your environment variable <usage-examples-env-variable>` to include 
:ref:`your connection string <usage-examples-connection-string>`. 
```